### PR TITLE
Upgrade license

### LINKED ISSUE #244: Upgrade to Creative Commons 4.0 license
We are using version 3.0. We should consider upgrading to 4.0: https://creativecommons.org/licenses/by-sa/4.0/legalcode

This involves changing license.txt and updating the release announcement.

We will need Dave's input on this one.

### PR DESCRIPTION
**Pull request by rjyounes**
Closes #244.

@Jamie-SA After the release, can you update the license reference on the website to this?

```
<a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>. In addition,  we require that any gist concepts remain in the gist namespace (https://w3id.org/semanticarts/ns/ontology/gist/), and that you not define your own terms within the gist namespace.
```

### CODE REVIEW

File: ontologies/gistCore.ttl [hunk 1]
```diff
@@ -14,7 +14,7 @@
 	owl:versionIRI <https://w3id.org/semanticarts/ontology/gistCoreX.x.x> ;
 	skos:definition "gist is a minimalist upper ontology created by Semantic Arts."^^xsd:string ;
 	skos:prefLabel "gist"^^xsd:string ;
-	gist:license "https://creativecommons.org/licenses/by-sa/3.0/"^^xsd:string ;
+	gist:license "https://creativecommons.org/licenses/by/4.0/"^^xsd:string ;
 	.
```
Comment (uscholdm) [inline ontologies/gistCore.ttl:18]: what is significance of /by/ vs. /by-sa/ what do either of them mean?

**Comment by Jamie-SA** [comment]
@rjyounes I've added a separate Issue wrt updating the website and assigned it to myself.

File: ontologies/gistCore.ttl (hunk 1 above)
Comment (rjyounes) [reply ontologies/gistCore.ttl:18]: I haven't looked into it. This was the license Dave chose based on answering the questions in the license chooser they provide, and I figure that's up to him.

Comment (dylan-sa) [reply ontologies/gistCore.ttl:18]: It looks like that segment of the URL is a code for the particular license. 'by' means 'Attirbution' and 'sa' in means 'sharealike'. 

More details here: https://creativecommons.org/licenses/

**Review by dylan-sa — APPROVED** [review]

**Review by dylan-sa — DISMISSED** [review]
@rjyounes -- The readme references 3.0, so we should update that too. Otherwise looking good to me.

**Review by uscholdm — APPROVED** [review]

**Comment by rjyounes** [comment]
> The readme references 3.0, so we should update that too. Otherwise looking good to me.

Wow, great catch, I don't know how you found that. Fixed.

**Comment by dylan-sa** [comment]
@rjyounes Sheer luck -- I was looking for the link to the gist-doc repo and happened to see it 😄

**Pull request merged**